### PR TITLE
fix: add parameter to the rule

### PR DESCRIPTION
**Pull request by saikiranakula-amzn**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by jonlouie — APPROVED** [review]

File: recommendation/microsoft.owin.hosting.json [hunk 1]
```diff
@@ -205,14 +208,17 @@
...
+              "Value": {
+                "comment": "Replace Microsoft.Owin.Hosting.WebApp.Start with WebHostBuilder such as: var host = new WebHostBuilder().UseKestrel().UseUrls(URL_HERE).UseStartup<Startup>().Build(); host.Start(); UseStartup can contain the TStartup class used before.",
+                "dontUseCTAPrefix": ""
+              },
               "Description": "Add a comment to explain how to replace WebApp.Start.",
               "ActionValidation": {
-                "Contains": "Replace Microsoft.Owin.Hosting.WebApp.Start with WebHostBuilder such as: var host = new WebHostBuilder().UseKestrel().UseUrls(URL_HERE).UseStartup<Startup>().Build(); host.Start(); UseStartup can contain the TStartup class used before.",
-                "NotContains": "",
-                "CheckComments": "True"
+                  "Contains": "Replace Microsoft.Owin.Hosting.WebApp.Start with WebHostBuilder such as: var host = new WebHostBuilder().UseKestrel().UseUrls(URL_HERE).UseStartup<Startup>().Build(); host.Start(); UseStartup can contain the TStartup class used before.",
```
Comment (Eruanion) [inline recommendation/microsoft.owin.hosting.json:217]: indent here got pushed out a bit can we match it with others?

File: recommendation/microsoft.owin.infrastructure.json [hunk 2]
```diff
@@ -117,14 +117,17 @@
...
+                "dontUseCTAPrefix": ""
+              },
               "Description": "Add a comment to explain how to replace the AddQueryString(string, string) invocation.",
               "ActionValidation": {
-                "Contains": "Please replace your parameters as the AddQueryString(string, string) method has been deprecated and you must either use the AddQueryString(string, string, string) method or the AddQueryString(string, System.Collections.Generic.IDictionary<string, string>) method.",
-                "NotContains": "",
-                "CheckComments": "True"
-              }
-            },
+                  "Contains": "Please replace your parameters as the AddQueryString(string, string) method has been deprecated and you must either use the AddQueryString(string, string, string) method or the AddQueryString(string, System.Collections.Generic.IDictionary<string, string>) method.",
```
Comment (Eruanion) [inline recommendation/microsoft.owin.infrastructure.json:126]: indent here as well.

File: recommendation/microsoft.owin.json [hunk 3]
```diff
@@ -703,14 +703,17 @@
...
+              "Value": {
+                "comment": "Please replace Get with TryGetValue.",
+                "dontUseCTAPrefix": ""
+              },
               "Description": "Add a comment to explain how to replace IOwinContext.Request.Headers.Get with TryGetValue instead.",
               "ActionValidation": {
-                "Contains": "Please replace Get with TryGetValue.",
-                "NotContains": "",
-                "CheckComments": "True"
+                  "Contains": "Please replace Get with TryGetValue.",
```
Comment (Eruanion) [inline recommendation/microsoft.owin.json:712]: indent

File: recommendation/microsoft.owin.json [hunk 4]
```diff
@@ -759,14 +762,17 @@
...
+              "Value": {
+                "comment": "Please replace Get with Items. Such as (Casting_Type)context.Items[\"String_To_Search_For\"];",
+                "dontUseCTAPrefix": ""
+              },
               "Description": "Add a comment to explain how to replace IOwinContext.Get with HttpContext.Items instead.",
               "ActionValidation": {
-                "Contains": "Please replace Get with Items. Such as (Casting_Type)context.Items[\"String_To_Search_For\"];",
-                "NotContains": "",
-                "CheckComments": "True"
+                  "Contains": "Please replace Get with Items. Such as (Casting_Type)context.Items[\"String_To_Search_For\"];",
```
Comment (Eruanion) [inline recommendation/microsoft.owin.json:771]: indent

**Comment by saikiranakula-amzn** [comment]
> do you need to make change in cta to read the file change?

The corresponding change is already made in the CTA, these changes are to fix the issues caused by cta change

**Review by Eruanion — COMMENTED** [review]
Looks good except for indentation in a few places. Are all unit tests in CTA passing with all of these rule changes also?

**Review by jonlouie — APPROVED** [review]

**Review by huawenm — APPROVED** [review]

**Pull request merged**